homeworks/ihor.zapisochniy_IgorFromUA/GamesProject/scripts/games.js

```diff
@@ -0,0 +1,233 @@
+        game_url: 'https://www.mmobomb.com/open/drakensang-online',
+        genre: 'MMORPG',
+        platform: 'Web Browser',
+        publisher: 'Bigpoint',
+        developer: 'Bigpoint',
+        release_date: '2011-08-08',
+        profile_url: 'https://www.mmobomb.com/drakensang-online',
+    },
+    {
+        id: 380,
+        title: 'Dark Orbit Reloaded',
+        thumbnail: 'https://www.mmobomb.com/g/380/thumbnail.jpg',
+        short_description: 'Take part in huge intergalactic battles and take on the whole galaxy in DarkOrbit, the free-to-play browser-based space combat MMO from Bigpoint -- now in 3-D! Choose your faction and your ship, each with their own strengths, and take off into adventure!',
+        game_url: 'https://www.mmobomb.com/open/darkorbit',
+        genre: 'Shooter',
+        platform: 'Web Browser',
+        publisher: 'Bigpoint',
+        developer: 'Bigpoint',
+        release_date: '2006-12-11',
+        profile_url: 'https://www.mmobomb.com/darkorbit',
+    },
+    {
+        id: 1122,
+        title: 'MultiVersus',
+        thumbnail: 'https://www.mmobomb.com/g/1122/thumbnail.jpg',
+        short_description: 'Match up in 1v1, 2v2 co-op, or 4-person free-for-all modes in this free-to-play Smash-Style Brawler!',
+        game_url: 'https://www.mmobomb.com/open/multiversus',
+        genre: 'Fighting',
+        platform: 'PC (Windows)',
+        publisher: 'Warner Bros. Games',
+        developer: 'Player First Games',
+        release_date: '2022-07-19',
+        profile_url: 'https://www.mmobomb.com/multiversus',
+    },
+    {
+        id: 5,
+        title: 'Crossout',
+        thumbnail: 'https://www.mmobomb.com/g/5/thumbnail.jpg',
+        short_description: 'Trick out your ride and take to the post-apocalyptic roads for battle in Crossout, the free-to-play vehicular combat game from Gaijin Entertainment! Featuring a vehicle design system with endless customization and fast-paced, armor-crunching combat, Crossout offers high-octane excitement in brief and explosive matches.',
+        game_url: 'https://www.mmobomb.com/open/crossout',
+        genre: 'Shooter',
+        platform: 'PC (Windows)',
+        publisher: 'Targem',
+        developer: 'Gaijin',
+        release_date: '2017-05-30',
+        profile_url: 'https://www.mmobomb.com/crossout',
+    },
+];
+const games = mocGames.map((game) => {
+    const copyGame = { ...game };
+    copyGame.small_description = `${game.short_description.slice(0, 60)}...`;
+    return copyGame;
+});
+const CARDS_LIST = document.querySelector('[data-cards]');
+const FORM_BUTTON = document.querySelector('[data-filter-button]');
+function createGameCardStr(game) {
```

Comment:
Please add an empty line before every function declaration. It will improve readability of your code

Reply:
added an empty line before every function declaration